### ISSUE TITLE
Ability to delete stored files

### ISSUE THREAD

**Issue by michaelcamper** [StreamPot/StreamPot#40]
**Is your feature request related to a problem? Please describe.**
The FAQ on the website say it is possible to delete stored files. However, I cannot find any documentation on how to do so

**Describe the solution you'd like**
Update the documentation to explain how to delete stored files

**Describe alternatives you've considered**
-

**Additional context**
-

**Comment by jackbridger** [comment]
Yes good point, it's actually not implemented. We will work on this soon and come back to you.

**Comment by jackbridger** [comment]
I have now added this. Just need to update docs but you can do like... 

```js
await streampot.deleteAssets({ jobId: 1 });
```
And it will delete all the assets associated with the job
